README.md

```diff
@@ -60,7 +57,8 @@ more flexibility and saves significant memory.
 
 ### Running Unitest
 
-We use `shunit2` as our unittest framework. If you want to enable unittest, please fetch your source along with `shunit2` by placing:
+We use `shunit2` as our unittest framework. If you want to enable unittest,
```

Comment:
Break the long line into 2.  Note that at the end of the file (line 74) the textwidth has been indicated to be 79.